Clarify we're using an older version of webots (#663)
srobo/docs: simulator/setting_up_simulator.md

To install Python, you can download the latest version from the [Python website](https://www.python.org/downloads/). If you have already installed Python from a package manager, such as homebrew on MacOS, apt on Ubuntu, or the Windows store on Windows, you can skip this step.
![python download site]({{ site.baseurl }}/images/content/simulator/python-download.png)

We recommend using [**Python 3.11**](https://www.python.org/downloads/release/python-3119/) as it is the version which is used on your physical robot.

> To install Webots, you can download the latest version from the [Webots website](https://cyberbotics.com/#download), however for the virtual league, Webots [R2023b](https://github.com/cyberbotics/webots/releases/tag/R2023b) will be used.

Reviewer: We already mention R2023b above. I'd be tempted to just link directly to the specific version, rather than the latest too:

```suggestion
To install Webots R2023b, you can download it from the [Webots GitHub](https://github.com/cyberbotics/webots/releases/tag/R2023b). Other versions may work, but this version will be used for the Virtual League.
```

Use the default settings when installing Webots.

> ![webots download site]({{ site.baseurl }}/images/content/simulator/webots-download.png)

Reviewer: This image is now out of date, since we're not using that page anymore. I suspect the image can be deleted rather than needing to recreate it for GitHub.
